theme_avantgarde/views/images_library.xml

```diff
@@ -477,5 +477,10 @@ Check in images.scss, primary_variables.scss, main.scss and theme_common's mixin
     <field name="name">website.s_showcase_default_image</field>
     <field name="url">/theme_avantgarde/static/src/img/pictures/bg_image_07.jpg</field>
 </record>
+<record id="s_website_form_overlay_default_image" model="theme.ir.attachment">
+    <field name="key">website.s_website_form_overlay_default_image</field>
+    <field name="name">website.s_website_form_overlay_default_image</field>
+    <field name="url">/theme_avantgarde/static/src/img/pictures/bg_image_08.jpg</field>
```

Comment:
It appears pixellated maybe something more abstract such as `s_pricelist_boxed_default_background.jpg` ?